modules/polling/components/VotesByAddress.tsx

```diff
@@ -48,13 +49,15 @@ const VotesByAddress = ({ tally, poll }: Props): JSX.Element => {
           {votes ? (
             <>
               {votes.map((v, i) => (
-                <tr key={i}>
+                <tr key={i} style={{ fontSize: bpi < 1 ? '12px' : '16px' }}>
```

Comment:
We should stick to our theme UI variables for font sizes

Reply:
how can i do 12px or 16px ? @tyler17  @b-pmcg @adamgoth ?

Reply:
Reference the `fontSize` array here and use the index: https://github.com/makerdao/dai-ui/blob/master/packages/dai-ui-theme-maker-neue/lib/index.js

12px would be `fontSize: 1` and 16px would be `fontSize: 3` since those are their indexes in that array.

Reply:
thanks @adamgoth  ,